start-from-zero/Javascript/src/views/index/index.js

```diff
@@ -35,14 +84,17 @@ function redirectLoginToEvaluation(){
     loginButton.addEventListener('click',function(e){
         e.stopPropagation();
```

Comment:
Does stopPropagation does anything that brings value here?

Reply:
Not really. I did some research and this is what I learned -> "If you stop propagation on the click event you're just stopping the bubbling phase of the click event. This has no effect on the submit event."  Is this correct ?

Reply:
in our case I doubt we need to stop the propagation of any event. Just preventing the default should be enough.